### PR TITLE
Fixed a few typos in DataFrame docs

### PR DESCRIPTION
**Pull request by ccphillippi**

### CODE REVIEW

File: docs/formulas.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # The Formula, ModelFrame and ModelMatrix Types
 
-In regression model, we often want to describe the relationship between a response variable and one or more input variables in terms of main effects and interactions. To facilitate the specification of a regression model in terms of the columns of a `DataFrame`, the DataFrames package provides a `Formula` type, which is created by the `~` binary operator in Julia:
+In regression modeling, we often want to describe the relationship between a response variable and one or more input variables in terms of main effects and interactions. To facilitate the specification of a regression model in terms of the columns of a `DataFrame`, the DataFrames package provides a `Formula` type, which is created by the `~` binary operator in Julia:
```
Comment (alyst) [inline docs/formulas.md]: Would "analysis" be a more commonly used term?

**Comment by ViralBShah** [comment]
The nightlies are failing the tests. I suspect this is ok to merge.

File: docs/formulas.md (hunk 1 above)
Comment (ccphillippi) [inline docs/formulas.md]: @alyst I agree with that. Changed to "analysis".

**Comment by alyst** [comment]
@ccphillippi Sorry for pickiness, could you please also squash the 1st and the 3rd commits together? I'll merge the PR then.

**Comment by ccphillippi** [comment]
@alyst how's that?

**Comment by alyst** [comment]
@ccphillippi  thanks!

**Pull request merged**